src/scenes/ModulesRouter/CurrentTimeline.tsx

```diff
@@ -0,0 +1,103 @@
+import { Box, Grid, Typography, useTheme } from '@mui/material';
+
+import strings from 'src/strings';
+
+const CurrentTimeline = () => {
+  const theme = useTheme();
+
+  // TODO these will probably come from the BE, not sure if they will be attached to the project, or cohort, or some
+  // other data model, so for now they are hard coded.
+  const currentPhase = 1;
+  const phases = [
+    {
+      phase: 0,
+      title: 'Phase 0 - Due Diligence',
+      description: 'Review application, final due diligence reviewing documents from shortlisted cohort.',
+    },
+    {
+      phase: 1,
+      title: 'Phase 1 - Feasibility',
+      description:
+        'Attending modules & 1:1s, filling out Feasibility Study sections, completing deliverables. Possible site visit.',
+    },
+    {
+      phase: 2,
+      title: 'Phase 2 - PDD Writing & Registration',
+      description: 'PDA signed, PDD writing from FS information, PDD registered on Verra (Under Development & Full).',
+    },
+    {
+      phase: 3,
+      title: 'Phase 3 - Should not be visible',
+      description: 'Mock desription',
+    },
+    {
+      phase: 4,
+      title: 'Phase 4 - Should not be visible',
+      description: 'Mock desription',
+    },
+  ];
+
+  const currentPhaseIndex = phases.findIndex((phase) => phase.phase === currentPhase);
+  const displayPhases = phases.slice(currentPhaseIndex - 1, 3);
+
+  return (
+    <Box borderRadius={theme.spacing(1)} padding={theme.spacing(3)} bgcolor={theme.palette.TwClrBgSecondary}>
+      <Grid display={'flex'} flexDirection={'column'}>
+        <Grid item marginBottom={theme.spacing(2)}>
+          <Typography fontWeight={600}>{strings.CURRENT_TIMELINE}</Typography>
+        </Grid>
+        <Grid item>
+          <Grid display={'flex'} flexDirection={'row'} justifyContent={'space-between'} alignItems={'center'}>
+            {displayPhases.map((phase, index) => (
+              <>
+                {index > 0 ? (
+                  <Grid item key={index + 0.5}>
```

Comment:
Interesting key, is it intentional? If so, why `0.5`?

Reply:
This is because we are dynamically inserting additional objects _between_ the `displayPhases`. Since the keys needs to be unique, and for this iteration we are already using `index`, we use a number in between `index` and the next `index`. It could be any decimal, or even any character.

Reply:
Makes sense, thanks for the explanation.